docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md

```diff
@@ -112,13 +112,13 @@ To update the `namePrefix`, perform the following steps:
 
     > **Note:** The value should be a 3-5 character long string like `cntso`. Longer strings are not recommended as they may conflict with Azure resource name length restrictions.
 
-    > **Note:** The CI pipelines automatically removes `localToken_` section from the name when processing the tokens replacement.
+    > **Note:** The CI pipelines automatically removes the `localToken_` prefix from the name when processing the tokens replacement.
 
     > **Note:** We highly encourage you to use the 'Check namePrefix availability' script ([see the documentation here](./Getting%20started%20-%20Check%20NamePrefix%20availability)) to check if the intended resource name will be available, based on the provided prefix.
 
  For further information on the token replacement logic, please refer to the corresponding [Token replacement](./The%20CI%20environment%20-%20Token%20replacement) section.
 
- If you do not prefer to set the `namePrefix` locally for your CI environment. You can optionally set it as a GitHub Secret or add it to the ADO variable group as the key `TOKEN_NAMEPREFIX` with value `your name prefix value` and that will take precedence over the one specified in the `settings.yml`.
+ If you do not prefer to set the `namePrefix` locally for your CI environment. You can optionally set it as a GitHub Secret or add it to the ADO variable group as the key `TOKEN_NAMEPREFIX` with value `your name prefix value` and that will only take precedence if the `localToken_namePrefix` is left empty in the `settings.yml`.
```

Comment:
```suggestion
If you want to avoid that the `namePrefix` is displayed in plain text your CI environment's settings file, you can optionally set it as a GitHub Secret or add it to the ADO variable group as the key `TOKEN_NAMEPREFIX` with a custom value. This value is only applied if the `localToken_namePrefix` in the `settings.yml` is left empty.
```

Reply:
updated to 

 If you do not prefer to set the `namePrefix` locally for your CI environment. You can optionally set it as a GitHub Secret or add it to the ADO variable group as the key `TOKEN_NAMEPREFIX` and its value `your name prefix value`, and this is only applied if the `localToken_namePrefix` in the [settings.yml](https://github.com/Azure/ResourceModules/blob/main/settings.yml) is left empty.